crates/iota-core/src/authority/scorer.rs

```diff
@@ -30,45 +44,97 @@ impl Scorer {
         let committee_size = voting_power.len();
         match protocol_config.scorer_version_as_option() {
             None | Some(1) => {
+                // Local metrics count are always initialized as zero.
                 let current_local_metrics_count = Arc::new(VersionedScoringMetrics::new(
                     committee_size,
                     protocol_config,
                 ));
 
-                let (received_metrics, metrics_missing_from, current_scores, invalid_reports_count) =
+                let max_score = 2_u64.pow(16);
```

Comment:
Would it be better to define a const like 
`const MAX_SCORE: u64 = 2_u64.pow(16);` in the front of the code? Same for the scale_factor.